common/source/docs/common-understanding-altitude.rst

```diff
@@ -0,0 +1,82 @@
+hoving.. _common-understanding-altitude:
+
+===================================
+Understanding ALTITUDE in ArduPilot
+===================================
+
+ArduPilot documentation uses the word "altitude" often and in many different ways and contexts. Not all "altitudes" mean the same thing, and an good understanding of what altitude means in any given context is important to understand expected operation and behavior.
+
+
+.. image:: ../../../images/altitudes.jpg
+    :target: ../_images/altitudes.jpg
+
+
+Definitions of Altitude (ALT)
+=============================
+
+In the above diagram, various types of "altitude" are shown:
+
+- ASL: Altitude above (mean) Sea Level - the altitude of the vehicle expressed as distance above the mean seal level of the world.
+- AGL: Altitude above Ground Level: the altitude of the vehicle above whatever is directly below it.
+- AH: Altitude above Home: the altitude distance above the Home position's altitude. This is what is usually displayed in the ground station and OSD as the vehicle's altitude.
+- Terrain ALT: The height above sea level (asl) of a terrain position. This sometimes, but not always includes, any natural or man-made additions to the terrain's ground altitude.
+- Estimated ALT: The autopilot's estimation of vehicle altitude above sea level which is used internally by the vehicle's Altitude Controller to maintain or obtain the Target ALT in altitude controlled modes (Plane: FBWB/AUTO/GUIDED/CRUISE/LOITER/CIRCLE), Copter: ALTHOLD/CIRCLE/LOITER/GUIDED/AUTO/DRIFT)
```

Comment:
within copter we normally use the altitude above EKF origin as the basis for our altitude hold in all modes.

Reply:
fixed origin vs home....I think its non-confusing